README.md

```diff
@@ -53,7 +53,6 @@ With the following configuration options set as environment variables:
 * `LEDGER_ADMIN_USER` (default: `'admin'`) The admin account's username (an admin user can create/modify accounts).
 * `LEDGER_ADMIN_PASS` (default: none) The admin account's password.
 * `LEDGER_ADMIN_FINGERPRINT` (default: none) The admin account's TLS certificate fingerprint if using TLS Client Certificate Auth.
-* `LEDGER_UNIT_DB_URI` (default `'sqlite://'`) Database to use for running unit tests. Configured separately for safety.
```

Comment:
Maybe we could add a note to the LEDGER_DB_URI line about the LEDGER_UNIT_DB_URI env var